README.md

```diff
@@ -64,7 +64,7 @@ $ annie https://youtu.be/Gnbch2osEeo
 >
 > `$ annie 'https://...'`
 
-The `-i` option displays all available formats, information without downloading.
+The `-i` option displays all information of video in the supplied link without downloading.
```

Comment:
`all information` seems not suitable, how about `all available quality and formats`

I think this looks simpler:

```
The `-i` option displays all available quality and formats without downloading.
```